edgelet/provisioning/src/provisioning.rs

```diff
@@ -1183,8 +1189,8 @@ mod tests {
             .then(|result| match result {
```

Comment:
These `match` statements on the `Result` instances can be written like so:
```rust
assert_eq!(
  result.err().unwrap(),
  ErrorKind::ExternalProvisioning(ExternalProvisioningErrorReason::InvalidCredentialSource));
```

Reply:
`result.unwrap_err()`

Reply:
Done